docs/resources/client/index.md

```diff
@@ -0,0 +1,15 @@
+Resource Packs
+==============
+
+[Resource Packs][respack] allow for the customization of client resources through the `assets` directory. This includes textures, models, sounds, localizations, etc. Forge, and your mod, can also have resource packs. Any user can therefore modify all the textures, models, and other asetts defined within this directory.
+
+### Creating a Resource Pack
+Resource Packs are stored within the `assets` directory within your project's resources.
+Your mod can have multiple asset domains, since you can add or modify already existing resource packs, like vanilla's, forge's, or another mod's.
```

Comment:
`forge` should be capitalized:
```suggestion
Your mod can have multiple asset domains, since you can add or modify already existing resource packs, like vanilla's, Forge's, or another mod's.
```